database/aoai-proxy.sql

```diff
@@ -445,7 +449,8 @@ CREATE TABLE aoai.owner_catalog (
     resource_name character varying(64) NOT NULL,
     endpoint_key character varying(128) NOT NULL,
     active boolean NOT NULL,
-    model_type aoai.model_type NOT NULL
+    model_type aoai.model_type NOT NULL,
+    location character varying(64) DEFAULT ''::character varying NOT NULL
```

Comment:
Rather than `location` should it be `region`, just to match Azure terminology?

Reply:
I think location is right. It's how it's referred to in the OpenAI service as Location

![197-1977645_cropping-webcam-circle-masking-overlay-for-obs-by-megasimman-luz-verde-de](https://github.com/microsoft/azure-openai-service-proxy/assets/9853509/c9396bfa-36d3-40d4-8d61-9c60c83226e4)